tests/management/role/test_dual_write.py

```
@@ -238,7 +238,11 @@ def given_v1_role_on_test_workspaces(
         )
 
     def given_update_to_v1_role(
-        self, role: Role, default: list[str] = [], replicator: Optional[RelationReplicator] = None, **kwargs: list[str]
+        self,
+        role: Role,
+        default: list[str] = [],
```

Ruff (0.16.1)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)